app/views/shared/check_answers/_cash_payments.html.erb

```diff
@@ -23,7 +23,7 @@
         <%= row.with_value do %>
           <% if cash_transactions.for_transaction_type(category.id).any? %>
             <% cash_transactions.for_transaction_type(category.id).each do |ctx| %>
-              <strong><%= ctx.transaction_date.strftime("%B") %></strong>: <%= gds_number_to_currency(ctx.amount, precision: 2) %><br>
+              <%= "#{gds_number_to_currency(ctx.amount, precision: 2)} in #{ctx.transaction_date.strftime('%B')}" %> <br>
```

Comment:
i assume the changing of the order here comes from designs. I will 🙈 but the use of the word `in` means it should technically be a translation 😬